doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -14,32 +14,28 @@ Prepare your PCB
 
 First, you need to create your PCB for the nRF54H20 SoC.
 
-We highly recommend using the PCB layouts and component values provided by Nordic Semiconductor, especially for clock and power sources.
-However, if you plan to select your own power sources, consider the following limitations:
+We highly recommend using the PCB layouts and component values provided by Nordic Semiconductor, especially for clock and power sources, considering the following limitations:
 
 * The DC/DC inductor must be present on the PCB.
-  If the DC/DC power converter is to be used, the BICR value ``POWER.CONFIG.INDUCTOR`` must also be programmed to ``Present`` (``0``).
-
-  .. note::
-     Currently, there is a bug (LIL-9968) causing the nRF54H20 limited sampling SoC to not wake up from the System OFF if DC/DC is enabled.
-     Temporarily, the workaround is to not use the DC/DC power converter.
-
+* the DC/DC power converter must be present on the PCB and enabled.
+  Also, the ``inductor-present;`` node must be configured either in :file:`zephyr/boards/your_board/nrf54h20dk_bicr.dtsi` or where the section ``bicr: bicr@fff87b0`` is present in the devicetree.
 * For the lowest sleep power consumption, use a 32 KHz crystal.
-* The **P9** port (3 V) cannot be used with internal nor external pull-down resistors.
-* For optimal performance, the output impedance of the **P6** and **P7** ports configured in ``IOPORT.DRIVECTRLX.PY`` should match the PCB and external device pin impedance.
-  However, an existing limitation requires this configuration to be 50 ohms.
+* The **P9** port cannot be used with internal nor external pull-down resistors.
+* For optimal performance, the output impedance of the **P6** and **P7** ports should match the PCB and external device pin impedance.
 
 
 Prepare the configuration files for your custom board in the |NCS|
 ******************************************************************
 
+##TODO
+
 The nRF54H20 DK uses multiple board files for its configuration:
 
 * `ARM cores configuration files`_
 * `Risc-V cores configuration files`_
 
 You can use these files as a starting point for configuring your own custom board.
-The easiest way to do so, when creating a :ref:`Zephyr repository application <zephyr:zephyr-repo-app>`, is to create a copy of these folders under :file:`sdk-nrf-next/boards/arm/your_custom_board_name` (for the ARM configuration files) and :file:`sdk-nrf-next/boards/riscv/your_custom_board_name` (for the Risc-V configuration files), respectively.
+The easiest way to do so, when creating a :ref:`Zephyr repository application <zephyr:zephyr-repo-app>`, is to create a copy of these folders under :file:`sdk-zephyr/boards/your_vendor_prefix/your_custom_board_name`.
 
 .. caution::
    Do not modify the configuration files related to the Secure Domain (:file:`*_cpusec` in the ARM folder) and the System Controller (:file:`*_cpusys` in the Risc-V folder).
```

Comment:
I cannot see any configuration files related to the Secure Domain or the System controller in `sdk-zephyr/boards/nordic/nrf54h20dk`. I believe these are now kept in `sdk-secdom` and `sdk-sysctrl` respectively, so they should not interfere here.

Reply:
Also the `nRF54H20_common_SoC_files` link is no longer valid. I think that should be pointing to `sdk-zephyr/dts/common/nordic/`

Reply:
I think you should definitely add a reference to this page: https://docs.nordicsemi.com/bundle/ncs-latest/page/zephyr/hardware/porting/board_porting.html